### ISSUE TITLE
TIMES partial equilibrium properties not being appreciated

### ISSUE THREAD

**Issue by Antti-L** [experimentarium/etsap-tiam#30]
The [Objective Function](https://github.com/experimentarium/etsap-tiam/blob/main/docs/model_objective/objective-function.md#objective-function) section now says the following (emphasis added):

- "Despite the strengths of the TIAM model, it is criticized for two issues arise from its partial equilibrium perspective. Similar to other bottom-up models the energy-service demands in TIAM are projected exogenously. This means that the model **ignores feedback between energy-service demands and their prices**. Therefore, it is not able to fully address the role of energy-service demands."

On the contrary, the documentation (Part I, which is used as a reference in the section) gives e.g. the following statements (emphasis added):
- "The concept of total surplus maximization extends the direct cost minimization approach upon which earlier bottom-up energy system models were based. These simpler models had fixed energy service demands, and thus were limited to minimizing the cost of supplying these demands. In contrast, **the TIMES demands for energy services are themselves elastic to their own prices**, thus allowing the model to compute a bona fide supply-demand equilibrium. This feature is a fundamental step toward capturing the main feedback from the economy to the energy system."

In other words, according to the documentation, TIMES maximizes the total surplus (it even goes further, saying that in TIMES "_each economic agent maximizes its own profit or utility_"), and the **energy services are elastic to their own prices**.  I think the current description of the objective function does not quite appreciate the properties of the TIMES equilibrium, despite the documentation being used as a reference.

**Comment by olejandro** [comment]
@LippkauFelix

**Comment by olejandro** [comment]
@Antti-L I agree with your point. Please feel free to contribute a more accurate description if you have the time. I believe @LippkauFelix, the original contributor of the text, is currently busy working on other sections of the docs.
